pkg/packetfilter/fake/packetfilter.go

```diff
@@ -44,12 +44,12 @@ type PacketFilter struct {
 }
 
 var (
-	iphookChainTypeToTableType = [packetfilter.ChainTypeMAX]packetfilter.TableType{
+	iphookChainTypeToTableType = []packetfilter.TableType{
```

Comment:
maybe we should use map also here (same as your suggested change for pkg/packetfilter/iptables/iptables.go) ?

Reply:
I didn't change this one b/c I actually remove it with https://github.com/submariner-io/submariner/pull/2924 😄